src/os/bluestore/BitAllocator.cc

```diff
@@ -636,7 +631,7 @@ int64_t BitMapZone::alloc_blocks_dis(int64_t num_blocks, int64_t zone_blk_off, i
  */
 int64_t BitMapArea::get_span_size()
 {
-  return BITMAP_SPAN_SIZE;
+  return g_conf->bluestore_bitmapallocator_span_size;
```

Comment:
Looks good. You can do same for g_conf->bluestore_bitmapallocator_blocks_per_zone instead of accessing this g_conf member at places, having a static function will look cleaner.